.gitlab/.gitlab-ci.yml

```diff
@@ -81,12 +81,6 @@ code-style:
     - $CI_PROJECT_DIR/util/memora_retry.sh get tc-llvm
     - ./util/lint.sh
   needs: ["tc-llvm"]
-  rules:
-    - if: '$CI_MERGE_REQUEST_ID'
```

Comment:
Can't we keep this now that the clang format path is fixed?

Reply:
The linting script check is very fast, I think it's worth to enable the linting check on any branch